Code-the-Dream-School/dd-prac-team1-front
# Dev/teampage PRA-156

## One Line Description
Responsive Our Team page and footer

## Requirements

_What this code should do_

Our Team page shows 9 team members, their images and short "about" text, each member card is clickable to their personal github pages.
Footer should show in the bottom of the pages throughout the app.
footer icons click out to corresponding webpages.
footer nav buttons styled and properly routed.
Buttons changed to solid green in Homepage and Contact pages
Responsive design

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

See footer in laptop and mobile layouts, click on Meet the Team to see Our Team page, also check it in mobile version.

## UI/UX Outcome

_Attach a screenshot here_
![OurTeammobile](https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/109550788/302d6015-108c-4284-b3a9-86c587911d42)

![OutTeam_Footer_mobile](https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/109550788/5d45bd5e-8386-4184-849e-9bc10a461490)

![OurTeam827](https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/109550788/27e6944c-f2b6-49e1-be9e-eae54144098f)

## Checklist


- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
